options.py

```diff
@@ -111,3 +111,9 @@ def desktop_opts_from_args(args):
         **vars(runtime_opts_from_args(args)),
         with_llvm = args.with_llvm
     )
+
+
+def make_default_args():
```

Comment:
Missing the `opts` parameter.

Reply:
Oh right, I assumed it was global from `options.py`. I'll fix that.

Reply:
Should be fixed. Testing on the buildsystem to confirm.